File: src/main/java/com/example/demo/model/Question.java
package com.example.demo.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "question")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Question {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Integer qid;
    String text;
    String answer1;
    String answer2;
    String answer3;
    Integer correct;
    Integer typeid;
    Boolean active;
}

File: ProjektowanieBazDanychT1/Models/Question.cs
namespace ProjektowanieBazDanychT1.Models
{
    public class Question
    {
        public int Id { get; set; }
        public string QuestionText { get; set; }
        public string Answer1 { get; set; }
        public string Answer2 { get; set; }
        public string Answer3 { get; set; }
        public string Answer4 { get; set; }
        public string CorectAnswer { get; set; }
    }
}